### PR TITLE
Fix 404 links

### PR DESCRIPTION
**Pull request by RiccardoMasutti**

### CODE REVIEW

File: docs/why-wasabi/TransactionSurveillanceCompanies.md [hunk 1]
```diff
@@ -112,3 +111,4 @@ Whether the activities of transaction surveillance company actually help against
 - [BlockSeer](https://www.blockseer.com/)
 - [ScoreChain](https://www.scorechain.com/)
 - [Crystal](https://crystalblockchain.com/)
+- Coinvalidation
```
Comment (MaxHillebrand) [inline docs/why-wasabi/TransactionSurveillanceCompanies.md:114]: Got a link here too?
https://www.coinvalidation.com/ seems to be down...

Comment (RiccardoMasutti) [reply docs/why-wasabi/TransactionSurveillanceCompanies.md:114]: That's why I removed it

Comment (MaxHillebrand) [reply docs/why-wasabi/TransactionSurveillanceCompanies.md:114]: Is the company still active? Should we remove them altogether then?

Comment (RiccardoMasutti) [reply docs/why-wasabi/TransactionSurveillanceCompanies.md:114]: > Is the company still active? Should we remove them altogether then?

Can't find info about it...

Comment (MaxHillebrand) [reply docs/why-wasabi/TransactionSurveillanceCompanies.md:114]: They started back in 2013, but I don't think they are still around... 

well, good to keep the historical record. the PR is good.

**Pull request merged**